### DISCUSSION TITLE
a note may need to be added to Windows step 15 (partition manager) to restart QFIL

### DISCUSSION THREAD

**Discussion by sudseh** [pgale/lineage_15.1_Installation_TSV#13] [Ideas]
When I pulled the power to the ThinkSmart and restarted it, there was no Partition Manager option within the Tools menu in QFIL until I restarted QFIL (after downloading the XML file).

Would it be possible to add a """note""" or other comment on step 15 (in Windows) to the effect that QFIL may need to be restarted to see the Partition Manager option?

**Comment by rajeshkmr1** [comment]
The reason Partition Manager option does not display in the Tools menu is because QFIL.exe must have been executed from the downloaded folder directly. Instead, run QFIL from the Start menu, which will show the Partition Manager option.

**Reply by pgale** [reply]
Nice tip - thanks.

**Reply by datenzar** [reply]
I wasn't able to get the entry until I used a more recent version of QFIL (qfiltool.com/qfil-tool-v2-0-3-5).
